.github/workflows/ci.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
 
       matrix:
-        ruby: [2.7.2]
+        ruby: [{ name: 2.7, value: 2.7.2 }]
```

Comment:
The action takes care of using the latest patch version available. Do we still prefer to lock this by patch level?

Reply:
We will maintain this as we prefer to lock by patch level.